src/_compress.pyx

```diff
@@ -18,7 +18,9 @@ cimport libc.stdio
 cimport openmp
 
 from libc.stdlib cimport malloc, free, realloc
-from libc.stdint cimport uint8_t, uint32_t
+ctypedef unsigned char  uint8_t
+ctypedef unsigned int   uint32_t
```

Comment:
Which ambiguity? `ctypedef unsigned long int uint32_t` ?
I think that would be uint64_t not uint32_t then
Actually ms vc++ doesn't have stdint.h on windows so only manual `ctypedef ` could help